### PR TITLE
Lotus

### PR DESCRIPTION
**Pull request by TweetyDaBird**
Added rp2040_ce and minor updates

### CODE REVIEW

File: keyboards/tweetydabird/lotus58/keymaps/vial/rules.mk [hunk 1]
```diff
@@ -7,15 +7,15 @@ VIAL_ENABLE = yes
 ENCODER_MAP_ENABLE = yes
 
 # Reduce size on atmega32u4
-ifeq ($(strip $(CONVERT_TO)), )
+ifeq ($(strip $(MCU)), atmega32u4)
```
Comment (lesshonor) [inline keyboards/tweetydabird/lotus58/keymaps/vial/rules.mk:10]: Not sure why this change is being reverted. It will result in these features being disabled (and LTO turned on) by default on all Vial builds (see #684).

Comment (TweetyDaBird) [reply keyboards/tweetydabird/lotus58/keymaps/vial/rules.mk:10]: Mostly because I had a disagreement with the Git client. We are working it out.

**Comment by xyzz** [comment]
Closing as no updates over a month
